static/js/src/stripe/set-modal-info.js

```diff
@@ -0,0 +1,102 @@
+const productNames = {
+  "uai-essential-desktop": "UA Infra Essential Desktop",
+  "uai-standard-desktop": "UA Infra Standard Desktop",
+  "uai-advanced-desktop": "UA Infra Advanced Desktop",
+  "uai-essential-physical-server": "UA Infra Essential Physical Server",
+  "uai-standard-physical-server": "UA Infra Standar Physical Server",
+  "uai-advanced-physical-server": "UA Infra Advance Physical Server",
+  "uai-essential-virtual-server": "UA Infra Essential Virtual Server",
+  "uai-standard-virtual-server": "UA Infra Standard Virtual Server",
+  "uai-advanced-virtual-server": "UA Infra Advanced Virtual Server",
+};
+
+const cardBrandImages = {
+  visa: "832cf121-visa.png",
+  mastercard: "83a09dbe-mastercard.png",
+  amex: "91e62c4f-amex.png",
+};
+
+function getProductsString(productsString) {
+  const productSlugs = eval(productsString);
```

Comment:
Using `eval` is a security risk. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#Never_use_eval!

I'm not entirely sure what you're trying to do here but there has to be a better way!